docs/practices/security_sensitive_issues.md

```diff
@@ -0,0 +1,37 @@
+## Security sensitive issues
+
+As nearcore is open source, all of its issues and pull requests are also
+publicly tracked on github.  However, from time to time, if a security sensitive
+issue is discovered, those cannot be tracked publicly on github.  However, we
+should promote as similar a development process to work on such issues as
+possible.  To enable this, below is the high level process for working on
+security sensitive issues.
+
+1. There is a [private fork of
+   nearcore](https://github.com/near/nearcore-private) on github.  Access to
+   this repository is restricted to the set of people who are trusted to work on
+   and have knowledge about security sensitive issues pertaining to nearcore.
+
+   This repository can manually synced with the public nearcore repository using
+   the following commands:
+
+       git remote add nearcore-public git@github.com:near/nearcore
+       git remote add nearcore-private git@github.com:near/nearcore-private
+       git fetch nearcore-public
+       git push nearcore-private nearcore-public/master:master
```

Comment:
````suggestion
    ```  console
    $ git remote add nearcore-public git@github.com:near/nearcore
    $ git remote add nearcore-private git@github.com:near/nearcore-private
    $ git fetch nearcore-public
    $ git push nearcore-private nearcore-public/master:master
    ```
````

style issues: 

* stick to fenced code blocks
* use `console` as a language and a leading `$` to emphasize that this is an interactive shell command.